app/models/application_record.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
 class ApplicationRecord < ActiveRecord::Base
+  include ActiveStorageSupport::SupportForBase64
```

Comment:
I'm not sure if its the best to place this line here or just do it in the models that will use active record.

Because if you place it here, the models that don't need active record would still have the methods that this line adds.

Try to see examples of where other people put this line.

Reply:
I was investigating and I saw that all users add them there. 
It is also recommended to apply it there on the official website

https://github.com/rootstrap/active-storage-base64/blob/master/README.md